### ISSUE TITLE
Trying to use local fonts

### ISSUE THREAD

**Issue by swampthang** [quilljs/quill#1668]
Attempting to use local fonts in an Electron app so wanted to get this to work in a CodePen first. Here's the CodePen with my attempt. https://codepen.io/swampthang/pen/BdOEqj?editors=0110

The fonts in the select menu all contain Sans Serif as a label. The styling in the menu looks correct. 

Another thing I noticed is that I can type a word, select it and apply any font in the list to the selected word but it only works once in Chrome and seems to apply the style to the rest of the items in the toolbar. In FireFox it doesn't seem to exhibit the same "once-only" behavior. 

I'm using Mac OS X Yosemite but the fonts chosen should be available on pretty much any system.

Latest version of Chrome and FireFox.

Quill version 1.3.1 (from CDN)

I'm pretty sure I just don't understand how to set this up correctly. I've tried to emulate 2 different examples for custom fonts but with no luck.

**Comment by benbro** [comment]
Please follow the issue template. It's hard to understand what your issue is.

There is a [CodePen example ](https://quilljs.com/playground/#custom-fonts)for using custom fonts.

**Comment by swampthang** [comment]
Sorry, but, though I didn't use the actual issue template headings, I did include the information requested in the template. 

Here's a revised version: https://codepen.io/swampthang/pen/vJVJgV

I see where, by creating a separate toolbar container, I also have to include the html for all the other settings items in order to get all the other options. This looks correct to me and seems to be working now. The only thing I have experienced with this is occasionally, when using bubble, the editor doesn't appear. I haven't been able to determine what I do to replicate this yet.
